### ISSUE TITLE
Add support for VPCOptions in ElasticSearch

### ISSUE THREAD

**Issue by Hyunk3l** [cloudtools/troposphere#861]
hello 😄 
some days ago AWS announced support for VPCs in ElasticSearch service:
https://aws.amazon.com/es/blogs/aws/amazon-elasticsearch-service-now-supports-vpc/

Here is an example from official documentation:
```
POST /2015-01-01/es/domain
{
    "DomainName": "<DOMAIN_NAME>",
    "ElasticsearchVersion": "<VERSION>",
    "ElasticsearchClusterConfig": {
        "InstanceType": "<INSTANCE_TYPE>",
        "InstanceCount": <INSTANCE_COUNT>,
        "DedicatedMasterEnabled": "<TRUE|FALSE>",
        "DedicatedMasterCount": <INSTANCE_COUNT>,
        "DedicatedMasterType": "<INSTANCE_TYPE>",
        "ZoneAwarenessEnabled": "<TRUE|FALSE>"
    },
    ...
    "VPCOptions": {
        "SubnetIds": [
            "<SUBNET_ID>"
        ],
        "SecurityGroupIds": [
            "<SECURITY_GROUP_ID>"
        ]
    },
    ...
}
```
Is there any plan for adding support for VPCOptions?

Meanwhile, thanks for the fantastic job you are doing!

thanks!

**Comment by craigbruce** [comment]
Unfortunately, AWS hasn't added this the CloudFormation API, see http://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-elasticsearch-domain.html, so it isn't accessible to CloudFormation and by extension Troposphere.

To encourage them, please submit a feature request (the more they get, hopefully the quicker they will implement it).

**Comment by Hyunk3l** [comment]
I was looking in the wrong place http://docs.aws.amazon.com/elasticsearch-service/latest/developerguide/es-configuration-api.html#es-configuration-api-datatypes-vpcoptions 😢 
thanks @craigbruce

**Comment by cintiadr** [comment]
While this is not yet documented, I did get the impression that this should be already working (keep in mind I haven't tested).

**Comment by stevengssns** [comment]
Fyi, the documentation has been updated, and it works.

http://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-elasticsearch-domain.html

**Comment by Hyunk3l** [comment]
PR updated :) thanks 4 the spot!

**Comment by markpeek** [comment]
Fixed via 0a38d79bf38c26c9dd543565c296e5f85f849827
